rhizome/postgres/bin/configure

```diff
@@ -4,6 +4,11 @@
 require "json"
 require_relative "../../common/lib/util"
 
+if ARGV.count != 1
+  fail "Wrong number of arguments. Expected 1, Given #{ARGV.count}"
+end
+
+v = ARGV[0]
```

Comment:
Should we cast it to an integer anyway?

Reply:
I think we shouldn't. Although we use only the major part currently, version numbers can contain non-digit characters such as "16.1" or "16-rc"